.github/workflows/android.yml

```diff
@@ -0,0 +1,58 @@
+name: Testing polyPod in Android
```

Comment:
The problem I see with a new test is that we would be building the apps in three different places, two of them the expensive and unreliable macos. Could you please consider adding this to the existing iOS.yaml? It already uses a macos runner, it already does a build, so it might be what you're looking for.